components/steering_wheel/include/IO/IO_Digital.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * IO_Digital.h
+ * Digital IO Module Header
+ * @note To transfer this module to another system, the digitalInput_E and the
+ *      typedef'd IO_Digital_Flag_t must be modified to be congruent with the new system
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "HW_gpio.h" /**< Needed for GPIO_TypeDef */
+#include "Types.h"
+
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef enum
+{
+    SW1 = 0U,
+    SW2,
+    SW4,
+    SW5,
+    BTN_TOP_LEFT,
+    BTN_MIDDLE,
+    BTN_TOP_RIGHT,
```

Comment:
seems like the paddle switches are missing?

Reply:
Yes, I omitted them because they are handled in IO_Analog

Reply:
gotcha. Those will probably be digital for now, I added analog functionality as a new feature but we'll start with digital. Anyway doesn't really matter we can fix it later